app/Models/CheckoutAcceptance.php

```diff
@@ -21,7 +21,7 @@ public function routeNotificationForMail(): string
     {
         // At this point the endpoint is the same for everything.
         //  In the future this may want to be adapted for individual notifications.
-        return (config('mail.reply_to.address')) ? config('mail.reply_to.address') : '' ;
+        return Setting::getSettings()['alert_email'];
```

Comment:
This change might be an issue for installs that expect these notifications to go to the `mail.reply_to.address` they have set...

Reply:
Should this be `Setting::getSettings()->alert_email, since it's a property on the Settings object`? Also what happens if that value is blank?

Reply:
I think there is some confusion on the notifications settings page because the user can enable email alerts but not set the email they should be sent to without a validation error. (Before this PR they got routed to the reply_to address and the `alert_email` was ignored for the notifications included in this PR). 

Should we update validation on the settings page?
![CleanShot 2023-02-01 at 11 47 24](https://user-images.githubusercontent.com/1141514/216147839-00ce97cb-6d82-4d13-a78e-fdd35dbd99bf.png)